src/main/cpp/util/file_windows.cc

```diff
@@ -755,22 +755,29 @@ void SyncFile(const string& path) {
 }
 
 bool MakeDirectoriesW(const wstring& path, unsigned int mode) {
-  if (path.empty()) {
+  std::wstring abs_path;
+  std::string error;
+  if (!AsAbsoluteWindowsPath(path, &abs_path, &error)) {
+    BAZEL_DIE(blaze_exit_code::LOCAL_ENVIRONMENTAL_ERROR)
+        << "MakeDirectoriesW(" << blaze_util::WstringToString(path)
+        << ") : " << error;
+  }
+  if (abs_path.empty()) {
```

Comment:
when does this happen?

Reply:
If `path` itself was empty.
Shall I move this check above the conversion?

Reply:
Yes please.

Reply:
Done.